UI/src/view/point_asset/pointAssetLayer.js

```diff
@@ -14,7 +14,9 @@
       assetLabel = params.assetLabel,
       allowGrouping = params.allowGrouping,
       assetGrouping = params.assetGrouping,
-      authorizationPolicy = params.authorizationPolicy;
+      authorizationPolicy = params.authorizationPolicy,
+      hasTrafficSignReadOnlyLayer = params.hasTrafficSignReadOnlyLayer,
+      trafficSignReadOnlyLayer = params.trafficSignReadOnlyLayer;
```

Comment:
attention when merging, is like readOnlyLayer: TrafficSignReadOnlyLayer, on master